traitcuration/traits/tasks.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
```

Comment:
Wait... Aren't we using Python 3 for all of this? Why is this import from `__future__` necessary?

Reply:
Yeah it was a recommended solution for a problem I was having with Celery finding some dependencies, it will be removed in the final iteration of this PR

Reply:
Looks like imports from the `__future__` are still there? I can see them in the `traitcuration/__init__.py` and `traitcuration/celery.py` modules

Reply:
Oh my bad, I forgot I included them in other files besides `tasks.py`. I removed them and pushed the changes.